CONTRIBUTING.md

```diff
@@ -29,6 +29,9 @@ This is a rough outline of what a contributor's workflow looks like:
 - Make sure all tests pass, and add any new tests as appropriate.
 - Submit a pull request to the original repository.
 
+For detailed contribution flow instructions for Rook, refer to
```

Comment:
Instead of using an href, let's use the markdown. And I'd suggest the wording:
For detailed contribution instructions, refer to the [development flow](Documentation/development-flow.md).